passes/sat/sim.cc

```diff
@@ -2397,6 +2397,140 @@ struct VCDWriter : public OutputWriter
 	std::ofstream vcdfile;
 };
 
+struct AnnotateActivity : public OutputWriter
```

Comment:
What is the significance of inheriting from `OutputWriter`?

Reply:
It executes in the natural flow of the sim command, at the end, when writing all the output files. 
It minimizes the intrusive changes to the sim code.

Reply:
Got it! Good idea